src/installation.md

```diff
@@ -0,0 +1,38 @@
+# PREREQUISITES
```

Comment:
Too long.. Just list it as

* CUDA: (installation instruction link to nvidia official website)
* PyTorch: (link to pytorch.org)
* Rust compiler: (like to https://rustup.rs/)

Then show how to install in the `Installation` section is enough.

Reply:
fixed